apps/yapms/src/lib/components/modals/importmodal/ImportModal.svelte

```diff
@@ -0,0 +1,167 @@
+<script lang="ts">
```

Comment:
I find it a bit awkward to have this modal be the only thing on the /app/imported route with a blank background. Maybe this logic can be integrated directly into the homepage. And this modal can be used in the /app/country/map/year route, and cause a navigate to the /app/imported page.